webapp/channels/src/components/suggestion/emoticon_provider.test.jsx

```diff
@@ -57,9 +57,9 @@ describe('components/EmoticonProvider', () => {
         expect(results.map((item) => item.name)).toEqual([
             'thumbsup', // thumbsup is a special case where it always appears before thumbsdown
             'thumbsdown',
+            'thunder_cloud_and_rain',
```

Comment:
I'm pretty sure these changes are correct. In `EmoticonProvider`, we weren't calling `compareEmojis` properly before so the sort order was wrong